### DISCUSSION TITLE
Authentication key not received in my email

### DISCUSSION THREAD

**Discussion by bibek-waterflow** [codemagic-ci-cd/codemagic-docs#2884] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Go to login page and enter your email.
2. In the screen it says code magic sents an authentication key to the mail but we have not received any mail from codemagic 

### Expected results

Authentication key should be sent to the mail and using the same key we should be able to login

### Actual results

1. No email has been received.
2. Not able to login which disrupts our flow of work

**Comment by dam-ease** [comment]
Hello @bibek-waterflow,
Please can you check your spam/junk folders? If you have implemented any spam filters, consider whitelisting our domain [no-reply@codemagic.io](mailto:no-reply@codemagic.io).
Also, can you connect via the contact [form](https://codemagic.io/contact/) and share your email so we can look into this. Sharing via the form is so that your email isn't exposed publicly.

**Reply by dam-ease** [reply, marked as answer]
Hello @bibek-waterflow,
Thanks for reaching out, I got your message and email via the contact form.
Please can you try to login again now, you should receive the Authentication key.
